web/src/lib/components/RoundInitial.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+	import { getInitials } from '$lib/name-wranglers';
+
+	export let name;
+</script>
+
```

Comment:
Maybe in the future we will set a default font that centers the text in the circle?